lib/matplotlib/widgets.py

```diff
@@ -1844,11 +1901,28 @@ def set_active(self, index):
         if self.eventson:
             self._observers.process('clicked', self.labels[index].get_text())
 
+    def clear(self):
```

Comment:
I'm wondering whether this should be named `reset()`.

Pro `reset`: This is the correct terminology. You cannot clear a radio button group.
Pro `clear`: Consistency with other widget. There are a number of widgets with `clear` methods, but none with `reset`.